website/docs/addon_maya_ornatrix_artist.md

```diff
@@ -98,3 +68,18 @@ The cache family exports Ornatrix Alembic (.abc) which includes the Ornatrix dat
 ### Loading
 
 The **Ornatrix Cache** supports to load Ornatrix alembic (.abc), Vray Proxy (.abc) and Arnold Scene Source (.abc).
+
+## FAQ
+### How to connect an Ornatrix rig with a pointcache or animation?
+After loading the Ornatrix Rig and a pointcache to connect it to, you can use the AYON tools to connect it for you:
+
+1. Go to `AYON` -> `Manage...`
+2. In the inventory select the assets from oxrig and animation/pointcache product type.
+3. Right-click `Actions` -> `Connect Ornatrix Rig`
+The Ornatrix geometry will be connected the animation rig with the transform and mesh attributes.
+
+:::tip
+The connecting will occur by node name - so if it fails, make sure the node name for the geometry is the same across your animation caches and the ornatrix exports.
```

Comment:
```suggestion
The connection depends on the name of node. If it fails, you need to make sure the node name for the geometry is consistent across your animation caches and the Ornatrix exports.
```

Reply:
what about: 
```suggestion
The connection relies on the node's name. If it fails, ensure that the geometry's node name is consistent between your animation caches and the Ornatrix exports.
```

Reply:
I've applied the last suggestion. feel free to unresolve.
https://github.com/ynput/ayon-documentation/pull/237/commits/5df28b622d2dbf737f57502e6b492bda2b003790